api/v1beta1/frrconfiguration_types.go

```diff
@@ -26,19 +26,29 @@ type FRRConfigurationSpec struct {
 	// +optional
 	BGP BGPConfig `json:"bgp,omitempty"`
 
+	// +optional
+	Raw RawConfig `json:"raw,omitempty"`
 	// Limits the nodes that will attempt to apply this config.
 	// When specified, the configuration will be considered only on nodes
 	// whose labels match the specified selectors.
 	// When it is not specified all nodes will attempt to apply this config.
 	// +optional
 	NodeSelector metav1.LabelSelector `json:"nodeSelector,omitempty"`
+}
+
+type RawConfig struct {
+	// Sets the order with this configuration is appended to the
+	// bottom of the rendered configuration
+	Priority int `json:"priority,omitempty"`
```

Comment:
does this have a value for the user (other than us using this for "deterministic order"?)

Reply:
well the order (and having the control of what config lands) is not only for us.
They might want to define the rotuer in one config and the rules in another, or what not.
It makes sense to control the order.